lib/video/screenRecording/recorder.js

```diff
@@ -10,8 +10,7 @@ module.exports = function getRecorder(options, browser, baseDir) {
     return new AndroidRecorder(options);
   } else {
     if (options.firefox.windowRecorder) {
-      if (options.browser === 'firefox' &&
-          (os.platform() === 'win32' || os.platform() === 'linux')) {
+      if (options.browser === 'firefox') {
         return new FirefoxWindowRecorder(options, browser, baseDir);
       }
       throw new Error('firefox.windowRecorder is only supported in Firefox on Windows and Linux');
```

Comment:
I see that this line needs to be updated to not reference the OS restriction, but I can't figure out how to do that in the web interface.  I'll pull and push a new commit if the behaviour change is actually OK.

Reply:
It looks like Android is already covered above it, so we probably don't need any of the os checks.  However, we should move the if (options.browser=='firefox') before the windowRecorder check or -b chrome hits the error on line 16.